docs/release_process.rst

```diff
@@ -0,0 +1,76 @@
+Steps of the SMQTK Release Process
+==================================
+Three types of releases are expected to occur:
+- major
+- minor
+- patch
+
+See the ``CONTRIBUTING.md`` file for information on how to contribute features
+and patches.
+
+The following process should apply when any release that changes the version
+number occurs.
+
+Create and merge version update branch
+--------------------------------------
+
+Patch Release
+^^^^^^^^^^^^^
+1. Create a new branch off of the ``release`` branch named something like
+   ``release-patch-{NEW_VERSION}``.
+    - Increment patch value in ``VERSION`` file.
+    - Rename the ``docs/release_notes/pending_patch.md`` file to
+      ``docs/release_notes/v{VERSION}.md``, matching the value in the
+      ``VERSION`` file.
+2. Merge version bump branch into ``release`` and ``master`` branches
+
+Major and Minor Releases
+^^^^^^^^^^^^^^^^^^^^^^^^
+1. Create a new branch off of the ``master`` branch named something like
+   ``release-[major,minor]-{NEW_VERSION}``.
+    - Increment patch value in ``VERSION`` file.
+    - Rename the ``docs/release_notes/pending_release.md`` file to
+      ``docs/release_notes/v{VERSION}.md``, matching the value in the
+      ``VERSION`` file.
+
```

Comment:
There should be step 2 (and maybe 3) here as well.  You need to merge the branch you created into master and then hard reset the release branch to point to master.

Reply:
I did miss that! Thanks for the catch.